single-table-inheritance/src/main/java/com/iluwatar/SingleTableInheritance/App.java

```diff
@@ -0,0 +1,48 @@
+package com.iluwatar.SingleTableInheritance;
+
+import com.iluwatar.SingleTableInheritance.ClassObject.Car;
+import com.iluwatar.SingleTableInheritance.ClassObject.Freighter;
+import com.iluwatar.SingleTableInheritance.ClassObject.Train;
+
+/**Singletable inheritance pattern map
+ * each instance of class in an inheritance tree into a single table.
+ */
```

Comment:
Please add a little bit more verbose explanation

- What the pattern is
- How the example app implements it